v63005/u_inref/gtm8877.csh

```diff
@@ -0,0 +1,86 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+source $gtm_tst/com/copy_ydb_dist_dir.csh ydb_temp_dist
+setenv gtm_dist $ydb_dist
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+
+echo "# NO RESTRICTIONS"
+$ydb_dist/mumps -run zsystemfn^gtm8877
+$ydb_dist/mumps -run pipeopenfn^gtm8877
+echo ""
+
+
+# FILTER RETURNING -1
+cat > $ydb_dist/restrict.txt << EOF
+ZSYSTEM_FILTER:filterfn^gtm8877
```

Comment:
Can we have a test where ZSYSTEM_FILTER and PIPE_FILTER are specified but without the : or entryref. To see what happens.

Reply:
System treats it as a syntax error and subsequently restricts everything (which happens when there is a syntax error in the restrict.txt). With this, the pecking order of ZSYSTEM and PIPE_OPEN vs ZSYSTEM_FILTER and PIPE_FILTER gets invoked, and it treats it as an invalid operation rather than a filtered operation

Reply:
Thanks. I have made sure we fix the GTM-88877 release note (to indicate the M labelref is mandatory and not optional) when we release r1.24 with the GTM-8877 fixes.